src/views/network/Network.vue

```diff
@@ -446,7 +489,32 @@
         .close_menu .v-speed-dial__list {
             padding-bottom: 8px;
         }
+        .network_search {
+            background-color: var(--v-controls-darken1);
+            border: 1px solid #cbcbcb;
+            border-top: 0 none;
+            overflow: hidden;
+            margin: 0;
+            padding: 0 6px 8px 6px;
+            position: absolute;
+            top: 0;
+            display: inline-block;
+            z-index: 99;
+            white-space: normal;
+            right: 0;
+        }
+        .network_search_field {
+            font-size: inherit;
+            margin: 0;
+            line-height: inherit;
+            padding: 0 6px;
+            min-width: 17em;
+            vertical-align: top;
+            min-height: 1.8em;
+            box-sizing: content-box;
+        }
```

Comment:
This can also all be replaced by:
```scss
.network_search {
	padding: 0 6px 8px 6px;
	position: absolute;
	top: 0;
	right: 0;

	.network_search_field {
		padding: 0 6px;
		width: 17em;
	}
}
```